cardshowviewtakenpicturesview/src/main/java/br/com/stant/libraries/cardshowviewtakenpicturesview/utils/ImageGenerator.java

```diff
@@ -105,15 +104,7 @@ private File createTempImageFileToShow(Bitmap bitmap, Integer typePhoto, Integer
         return file;
     }
 
-    private void saveInPictures(Bitmap bitmap, Integer orientation, String uuid){
-        try {
-            subscribeSaveImageInPicturesThread(rotateImage(bitmap, orientation), uuid);
-        } catch (OutOfMemoryError outOfMemoryError) {
-            subscribeSaveImageInPicturesThread(bitmap, uuid);
-        }
-    }
-
-    private void subscribeSaveImageInPicturesThread(Bitmap bitmap, String uuid) {
+    public void subscribeSaveImageInPicturesThread(Bitmap bitmap, String uuid) {
```

Comment:
Move subscribeSaveImageInPicturesThread method to ImageDecoder to make it globally available. See the importance of the saveInPictures method that rotates images before save, maybe its important maintain the method for future uses or rollback of the function.